src/plugins/discover/public/application/components/data_grid/data_grid_table_cell_value.tsx

```diff
@@ -58,12 +59,12 @@ export const fetchTableDataCell = (
     return <span>-</span>;
   }
 
-  if (!fieldInfo?.type && flattenedRow && typeof flattenedRow[columnId] === 'object') {
+  if (!fieldInfo?.type && typeof flattenedRow?.[columnId] === 'object') {
```

Comment:
does javascript end up equating this as undefined and null but then typeof of null equating to an object?

Reply:
`flattenedRow?.[columnId]` would be `undefined` if `flattenedRow` was `null`, `undefined`, `Boolean(false)`, `Number(0)`, or `""`. For the last 3, the prototype props and functions can be named in `columnId` but then, they could be called with any `Boolean`, `Number`, or `String` and if we wanted to block those from being used, we would need to handle them for all of those types and not just for the falsy ones.